packages/dev-utils/package.json

```diff
@@ -12,9 +12,13 @@
   "keywords": [
     "celo"
   ],
+  "files": [
+    "lib/*"
+  ],
   "scripts": {
```

Comment:
Added repack and explicit files as libs because one published version of this had src files